2021-11-24 23:45:41,949 INFO    MainThread:51024 [wandb_setup.py:_flush():71] setting env: {}
2021-11-24 23:45:41,950 INFO    MainThread:51024 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2021-11-24 23:45:41,950 INFO    MainThread:51024 [wandb_init.py:_log_setup():371] Logging user logs to /home/snamburi/Debarshi/wandb/run-20211124_234541-10j4bu8o/logs/debug.log
2021-11-24 23:45:41,950 INFO    MainThread:51024 [wandb_init.py:_log_setup():372] Logging internal logs to /home/snamburi/Debarshi/wandb/run-20211124_234541-10j4bu8o/logs/debug-internal.log
2021-11-24 23:45:41,950 INFO    MainThread:51024 [wandb_init.py:init():404] calling init triggers
2021-11-24 23:45:41,950 INFO    MainThread:51024 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-24 23:45:41,950 INFO    MainThread:51024 [wandb_init.py:init():449] starting backend
2021-11-24 23:45:41,951 INFO    MainThread:51024 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-24 23:45:41,983 INFO    MainThread:51024 [backend.py:ensure_launched():199] starting backend process...
2021-11-24 23:45:42,005 INFO    MainThread:51024 [backend.py:ensure_launched():204] started backend process with pid: 51034
2021-11-24 23:45:42,007 INFO    MainThread:51024 [wandb_init.py:init():458] backend started and connected
2021-11-24 23:45:42,009 INFO    MainThread:51024 [wandb_init.py:init():519] updated telemetry
2021-11-24 23:45:42,011 INFO    MainThread:51024 [wandb_init.py:init():549] communicating current version
2021-11-24 23:45:42,814 INFO    MainThread:51024 [wandb_init.py:init():554] got version response 
2021-11-24 23:45:42,815 INFO    MainThread:51024 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-24 23:45:43,008 INFO    MainThread:51024 [wandb_init.py:init():592] starting run threads in backend
2021-11-24 23:45:44,510 INFO    MainThread:51024 [wandb_run.py:_console_start():1816] atexit reg
2021-11-24 23:45:44,510 INFO    MainThread:51024 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-24 23:45:44,511 INFO    MainThread:51024 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-24 23:45:44,514 INFO    MainThread:51024 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-24 23:45:44,514 INFO    MainThread:51024 [wandb_init.py:init():619] run started, returning control to user process
2021-11-24 23:45:44,515 INFO    MainThread:51024 [wandb_run.py:_config_callback():962] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.5, 'amp': True}
2021-11-24 23:45:45,462 INFO    MainThread:51024 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-24 23:45:45,463 INFO    MainThread:51024 [wandb_run.py:_restore():1758] restore
2021-11-24 23:45:47,817 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 23:45:47,981 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 564
}

2021-11-24 23:45:48,105 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 80458
}

2021-11-24 23:45:48,211 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 564
  total_bytes: 80458
}

2021-11-24 23:45:48,314 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 80458
  total_bytes: 80458
}

2021-11-24 23:45:48,417 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 80458
  total_bytes: 80458
}

2021-11-24 23:45:48,520 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 80458
  total_bytes: 80458
}

2021-11-24 23:45:48,624 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 80458
  total_bytes: 80458
}

2021-11-24 23:45:48,727 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 80458
  total_bytes: 80458
}

2021-11-24 23:45:48,840 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 80458
  total_bytes: 80458
}

2021-11-24 23:45:49,104 INFO    MainThread:51024 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 80458
  total_bytes: 80458
}
local_info {
}

2021-11-24 23:45:50,274 INFO    MainThread:51024 [wandb_run.py:_append_files():2184] logging synced files
